types/mdast/index.d.ts

```diff
@@ -91,22 +201,22 @@ export interface Blockquote extends Parent {
 
 export interface List extends Parent {
     type: 'list';
-    ordered?: boolean | undefined;
```

Comment:
I believe the `| undefined`s are intentional. Or at least, will be added again soon: https://github.com/DefinitelyTyped/DefinitelyTyped/commit/08d3d0e71d834ff2f420ef2590d4aff97d25d042

Reply:
I restored them and disabled the tslint rule that reports them.